packages/core/src/schemas/publisherJsonMetadata.json

```diff
@@ -50,10 +50,12 @@
                   "type": "string"
                 },
                 "website": {
-                  "type": "string"
+                  "type": "string",
+                  "pattern": "^(https?://[^/]+)[/]{0,1}"
                 },
                 "contact": {
-                  "type": "string"
+                  "type": "string",
+                  "pattern": "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$"
```

Comment:
It's not common, and some email clients don't support all these characters, but the local part of the regex should probably be something like: ``[a-zA-Z0-9_!#$%&'*+/=?`{|}~^.-]``

Reply:
Can update